### DISCUSSION TITLE
Photo import name

### DISCUSSION THREAD

**Discussion by jeannotalpin** [photoprism/photoprism#2746] [General]
Hi,
There seem to be an issue in the title detection based on filename. Uploading images with the following filename creates an image with the following titles :
* DSF2001.jpg : Unknown / 2022
* Photo_mer.jpg : Unknown / 2022
* Photo_de_la_mer.jpg : Unknown / 2022
* Mer_photo.jpg : Mer Photo / 2022 
* Mer_2001.jpg : Unknown / 2022
* Mer_photo_2001.jpg : Mer Photo / 2022 

There seems to be a rule, but quite hard to decipher. For information, the images have many exif fields with various descriptive information, but those are not used for the title apparently.

IMO, there are 2 issues:
1. As a user I want to be able to deterministically set the title of uploaded pictures with the filename or via an exif field
2. As a user I want to know the title rules by reading the documentation. I haven't found this in the existing documentation

![image](https://user-images.githubusercontent.com/480536/193423295-8a2e1172-9d3b-4a8e-b80b-b6efa2a01a79.png)

I am using the demo server (Build 220919-cc8bab446) but I have the same issue on my instance (Build 220629-5d7448d2).

**Comment by graciousgrey** [comment]
The title generation works as designed.

In case there is no title information in the files metadata (https://photoprism.app/kb/metadata), titles are generated dynamically based on detected countries, labels or keywords.  The title is not meant to display the filename. 

Offering settings to adapt the title generation to individual needs is on our to do :)
